plans/tier1.fmf

```diff
@@ -285,3 +285,11 @@ adjust+:
         discover+:
             test+<:
                 - proxy-conversion/proxy_conversion
+
+    /error-after-ponr:
+        discover+:
```

Comment:
Just an idea but maybe we could use the "nonplus" syntax, which may indicate more that we are overriding something.
```suggestion
        discover:
          how: fmf
          test:
            - error-after-ponr
```

Reply:
Changing to "nonplus" `discover` then forces to add `how: fmf` every time and probably it will still have the "nonplus" `test`. Although it helps readability I think a code comment like the one there is now already works.

Reply:
<img src="https://media0.giphy.com/media/3oz8xQQP4ahKiyuxHy/giphy.gif"/>